dispatch/dispatch.go

```diff
@@ -0,0 +1,60 @@
+package dispatch
+
+import (
+	. "fmt"
+	"os"
+
+	"polydawn.net/repeatr/def"
+	"polydawn.net/repeatr/executor"
+	"polydawn.net/repeatr/executor/null"
+	"polydawn.net/repeatr/input"
+	"polydawn.net/repeatr/input/dir"
+	"polydawn.net/repeatr/input/tar"
+	"polydawn.net/repeatr/output"
+)
+
+// TODO: This should not require a global string -> class map :|
+// Should attempt to reflect-find, trying main package name first.
+// Will make simpler to use extended transports, etc.
```

Comment:
I'm thinking ahead here to outside-this-repository implementations. 

We can totally maintain a string mapping until the end of time, and you're right in that package strings might not always be a ceiling here, but I'd like to be able to put

```json
{
	"Type": "not.your.domain.com/lol/wat",
	"Location": "/var/log",
	"URL": "whelp.tar"
}
``` 

And have that Just Work. 

Or to clarify further, scitran engine will be rolling our own scheduler and almost certainly some custom transports; importing most of repeatr and interoperating it would be nice.

A reflect lookup could be added as a final lookup, post-strings.